src/aclib/ac_utils/ac_utils.cpp

```diff
@@ -148,20 +151,15 @@ args_t ac_init_args( int ac, char* av[]){
             av++;
         }
 #ifdef USE_GDB
-        //     if( (size>=5) && (!strncmp( av[1], "--gdb", 5))){ //Enable GDB support
-        //       int port = 0;
-        //       if ( size > 6 )
-        //         {
-        //           port = atoi( av[1] + 6 );
-        //           if ( ( port > 1024 ) && gdbstub )
-        //             gdbstub->set_port( port );
-        //         }
-        //       if ( gdbstub )
-        //         gdbstub->enable();
-        //     }
-#endif /* USE_GDB */
+        else if ((size >= 7) &&
```

Comment:
In this if-else statement, the 'atoi' function converts the string after '=' in number. If the string is not a valid integral number, or if no such sequence exists because either str is empty or it contains only whitespace characters, no conversion is performed and zero is returned. Port below 1024 is ignored and the default port, 5000, is used.